.github/workflows/generative-tests.yml

```diff
@@ -22,3 +22,13 @@ jobs:
           GENTEST_DEBUG=t \
           GENTEST_CHECK_IGNORED_ERRORS=t \
           just test-generative
+
+      - name: "Notify Slack on Failure"
+        if: failure()
+        uses: voxmedia/github-action-slack-notify-build@3665186a8c1a022b28a1dbe0954e73aa9081ea9e
```

Comment:
Might be useful to put the version number after this to save us searching in the future, we've done `  # vN.N.N` elsewhere and dependabot typically updates that too!

Reply:
Oh nice! I'll do that now